mmv1/templates/terraform/metadata.yaml.tmpl

```diff
@@ -0,0 +1,5 @@
+resource: '{{ $.TerraformName }}'
+generation_type: 'mmv1'
+api_name: '{{ $.ProductMetadata.ServiceName }}'
```

Comment:
Should we change this to `api_service_name` to match go/unified-resource-type?

Reply:
yeah that seems reasonable to me